rpc/utils.py

```diff
@@ -0,0 +1,27 @@
+import re
+import string
+import random
+
+
+def create_new_address() -> str:
```

Comment:
@sevenearths these two methods about the address look like they should be at the protocol level. @jmlago do you think they should be in the GenVM?